SPECS/cloud-init/add-mariner-distro-support.patch

```diff
@@ -372,6 +371,16 @@ index c170aef7..20ff1daa 100644
  DefaultDependencies=no
  {% endif %}
  Wants=cloud-init-local.service
+@@ -26,8 +26,8 @@
+ Before=network-online.target
+ Before=sshd-keygen.service
+ Before=sshd.service
+-{% if variant in ["ubuntu", "unknown", "debian"] %}
+ Before=sysinit.target
++{% if variant in ["ubuntu", "unknown", "debian"] %}
```

Comment:
While this will certainly patch it fine for Mariner, I'd just ask to keep consistent with the variant checks, e.g.:

```suggestion
+{% if variant in ["ubuntu", "unknown", "debian", "mariner"] %}
```

Reply:
But regardless, feel free to leave it as-is and I'll just pay close attention to the deltas when upstreaming :D

Reply:
The reason I am not adding mariner is because it will also add shutdown.target to the change, which I am not sure if necessary